BasicGates/ReferenceImplementation.qs

```diff
@@ -96,44 +81,44 @@ namespace Quantum.Kata.BasicGates {
     //        If the qubit is in state |1⟩, change its state to exp(i*alpha)|1⟩.
     //        If the qubit is in superposition, change its state according to the effect on basis vectors.
     operation PhaseChange_Reference (q : Qubit, alpha : Double) : Unit
```

Comment:
This is a good time to swap the arguments in this operation, since you're editing the signature of the method anyways?

Reply:
Sounds good, happy to do so.